[2025-08-28T08:27:39.597+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_pipeline.archive manual__2025-08-28T08:27:02.896782+00:00 [queued]>
[2025-08-28T08:27:39.616+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_pipeline.archive manual__2025-08-28T08:27:02.896782+00:00 [queued]>
[2025-08-28T08:27:39.618+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2025-08-28T08:27:39.656+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): archive> on 2025-08-28 08:27:02.896782+00:00
[2025-08-28T08:27:39.704+0000] {standard_task_runner.py:60} INFO - Started process 622 to run task
[2025-08-28T08:27:39.713+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'etl_pipeline', 'archive', 'manual__2025-08-28T08:27:02.896782+00:00', '--job-id', '15', '--raw', '--subdir', 'DAGS_FOLDER/etl_pipeline.py', '--cfg-path', '/tmp/tmpo7zyo4je']
[2025-08-28T08:27:39.750+0000] {standard_task_runner.py:88} INFO - Job 15: Subtask archive
[2025-08-28T08:27:39.836+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:194 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-08-28T08:27:40.750+0000] {task_command.py:423} INFO - Running <TaskInstance: etl_pipeline.archive manual__2025-08-28T08:27:02.896782+00:00 [running]> on host dd18df0f367e
[2025-08-28T08:27:41.344+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_pipeline' AIRFLOW_CTX_TASK_ID='archive' AIRFLOW_CTX_EXECUTION_DATE='2025-08-28T08:27:02.896782+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-08-28T08:27:02.896782+00:00'
[2025-08-28T08:27:41.353+0000] {logging_mixin.py:188} INFO - Archiving (fake)...
[2025-08-28T08:27:43.359+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-08-28T08:27:43.708+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=etl_pipeline, task_id=archive, execution_date=20250828T082702, start_date=20250828T082739, end_date=20250828T082743
[2025-08-28T08:27:44.189+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-08-28T08:27:44.607+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
